src/sir.cpp

```diff
@@ -1,4 +1,4 @@
-// Generated by dust (version 0.5.8) - do not edit
+// Generated by dust (version 0.5.10) - do not edit
```

Comment:
This version is wrong, but I don't know if it matters (is it always one behind?)

Reply:
I should regenerate again; probably I updated it before increasing the version number